### PR TITLE
Expansions on taxation

### PR DESCRIPTION
**Pull request by jlevy**
And a few other ongoing improvements.

### CODE REVIEW

**Comment by jlevy** [comment]
Cool. Glad you agree on the direction! More big comments/suggestions welcome; was editing quickly and still more to come so careful style and copy edits not needed just yet.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 - [Taxes on RSUs](#taxes-on-rsus)
 - [Stages of a Startup](#stages-of-a-startup)
 - [Evaluating Equity Compensation](#evaluating-equity-compensation)
-- [Negotiations and Offers](#negotiations-and-offers)
+- [Negotiations and Offers](#offers-and-negotiations)
```
Comment (RacheltheEditor) [inline README.md]: Link text: Offers and Negotiations

File: README.md [hunk 2]
```diff
@@ -234,29 +249,37 @@ And what is this ownership worth?
...
+  equity turning into large amounts of cash in the future.
+- To align individuals’ incentives with the interests of the company:
+  Even companies that can afford to pay lots of cash may prefer to give employees equity, so
+  that employees work to increase the *future* value of the company.
+  At its best, this philosophy fosters an environment of teamwork and a “rising tides lift
+  all boats” mentality.
+  As we’ll [discuss later](#offers-and-negotiations), the amount of equity you’re offered
+  usually reflects both your past or current contribution to the company and your commitment
+  to the company in the future.
+- To reduce the company’s spending on cash compensation:
```
Comment (RacheltheEditor) [inline README.md]: Simon (Dummies) refers to 'cash diversion.' We may want to 🌪 this and talk a bit about the potential for abuse of this tactic? And controversy of skyrocketing equity-based pay for CEOs (data and interesting figures in this 2013 study: https://www.haygroup.com/downloads/us/wsj%20hay%20group%202013%20ceo%20compensation%20study.pdf. That year, 37% CEO pay is in cash, 54% in stock and options. Equity pay increased after 2008 financial crises, connecting us back to diverted cash.

File: README.md [hunk 3]
```diff
@@ -562,9 +585,9 @@ common for startups.
 contracts that allow you to buy shares in the company you work for.
 When you buy shares, you are **exercising** the options.
 
-- 🔸 Options are not the same as stock;
-  they are only the right to buy stock upon and subject to the conditions specified in the
-  option agreement.
+☝️ Options are not the same as stock;
```
Comment (RacheltheEditor) [inline README.md]: Even more clarity: 'Stock options are just that: the _option to buy stock_ at a certain price (the strike price) on a certain schedule (the vesting schedule). If you have stock options, you are not a shareholder until you _exercise_ (or purchase) your stock. Now we'll dive into the details of all of that.'

**Comment by jlevy** [comment]
Merging so I can look at first parts of structural changes separately.

File: README.md (hunk 2 above)
Comment (jlevy) [reply README.md]: If we open an evaluation of this it probably requires more than that single link on CEO compensation, which is just one aspect. I'll track this issue separately.

**Pull request merged**